sklearn/preprocessing/_data.py

```diff
@@ -2262,6 +2262,8 @@ def _dense_fit(self, X, random_state):
                 col = col.take(subsample_idx, mode='clip')
             self.quantiles_.append(np.nanpercentile(col, references))
         self.quantiles_ = np.transpose(self.quantiles_)
+        self.quantiles_ = np.maximum.accumulate(
+                                    self.quantiles_)
```

Comment:
1. no need to skip line (unless it's too long?)

2. maybe add a small comment e.g.

`# make sure quantiles are monotonically increasing`

Reply:
Done!